adagrams/game.py

```diff
@@ -1,13 +1,80 @@
 from random import randint
-
+letters_pool={"A":9,"B":2,"C":2,"D":4,"E":12,"F":2,"G":3,"H":2,"I":9,"J":1,"K":1,"L":4,"M":2,"N":6,"O":8,"P":2,"Q":1,"R":6,"S":4,"T":6,"U":4,"V":2,"W":2,"X":1,"Y":2,"Z":1}
 def draw_letters():
-    pass
+    drawed_letters=[]
+    all_letters=[]
+    letters_pool={"A":9,"B":2,"C":2,"D":4,"E":12,"F":2,"G":3,"H":2,"I":9,"J":1,"K":1,"L":4,"M":2,"N":6,"O":8,"P":2,"Q":1,"R":6,"S":4,"T":6,"U":4,"V":2,"W":2,"X":1,"Y":2,"Z":1}
```

Comment:
Fix up inconsistent white spacing around operators and use multilines for when your line of code exceeds 79 characters. 
```suggestion
    drawed_letters = []
    all_letters = []
```

You've already created the dictionary with letters and their frequencies on line 2. However, line 6 repeats what's already written on line 2. Whenever possible, we should eliminate repetition to keep our code DRY (Don't Repeat Yourself). If you're not familiar with DRY, do some research to review why it's important for maintaining readable codebases. 

Since you already created `letters_pool` on line 2, you can remove the repeated one on line 6 and just use your global variable. 

However, notice that the only function that uses `letters_pool` is `draw_letters`. Therefore, we **don't** need this constant variable to have global scope. We could only create the variable inside this method, remove the global variable and confine the scope to this method (which is preferred). Variables should have the most restrictive scope as possible and only when we need really need to, should we make a variable global.